README.md

```diff
@@ -23,11 +23,13 @@ Questions or feedback? Join us at [RocketChat](https://community.numer.ai/channe
 
 All you need is:
 
-1.  AWS (Amazon Web Services) account setup with an API key
-2.  A Numerai API key
-3.  Docker setup on your machine
-4.  Python3 (your model code doesn't have to use Python3, but this CLI tool needs it)
+1.  AWS (Amazon Web Services) account setup 
+2.  AWS API keys (Access key ID and Secret Access Key)
+3.  Numerai API Key (Public ID and secret key)
+4.  Docker setup on your machine
+5.  Python3 (your model code doesn't have to use Python3, but this CLI tool needs it)
 
+For OSX, you can download and run `scripts/setup-mac.sh` to setup your computer for this tool.
```

Comment:
should we also mention the windows/linux scripts?

Reply:
yes, just pushed updates to instructions